Studs-App/stud-client
# Created a Table for the Profile Page

**What I Did:**

- Added an expanding table that can take data from our backend.
- For now, it's using mock data until the backend is set up.

**How To Test:**

- Have Vue.JS set up and run `npm install` if needed. 

- Visit `localhost:8080` to test, the table should be there and you should be able to expand each row.
